crypto/fipsmodule/cipher/e_aes.c

```diff
@@ -1541,6 +1542,94 @@ static int aead_aes_gcm_tls13_seal_scatter(
   return 0;
 }
 
+#define AEAD_AES_GCM_TLS13_STATE_SERDE_VERSION 1
+
+/*
+ * AeadAesGCMTls13StateSerializationVersion ::= INTEGER {v1 (1)}
+ *
+ * AeadAesGCMTls13State ::= SEQUENCE {
+ *   serializationVersion AeadAesGCMTls13StateSerializationVersion,
+ *   minNextNonce   [0] INTEGER,
+ *   mask           [1] INTEGER,
+ *   first          [2] BOOLEAN
```

Comment:
Looks like I forgot to fix this comment, these should just be `INTEGER` and `BOOLEAN` respectively. No tag.